composer.json

```diff
@@ -14,7 +14,7 @@
     "require": {
         "php": "^5.3.9|^7.0",
         "symfony-cmf/routing": "~1.2",
-        "symfony/framework-bundle": "~2.3"
+        "symfony/framework-bundle": "~2.3|~3.0"
```

Comment:
as we're now using the `placeholder` option, 2.3 has to be 2.6. As 2.6 is no longer supported, let's make it 2.7.